src/backend/types/write-only-stream.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for dns-discovery 6.2.3
+// Project: https://github.com/mafintosh/dns-discovery
+// TypeScript Version: 4.2
+
+/// <reference types="node" />
+
+declare module "write-only-stream" {
```

Comment:
What is this used for?

Reply:
That is now unused. I used it in my first attempt at `beforeAfterStream.js`.